Add EdgeBoard.md

zuoyekun opened the pull request:
## Create EdgeBoard.md

will-jl944 committed:
add restful

will-jl944 committed:
update html demo

will-jl944 committed:
fix export bug

will-jl944 committed:
fix restful web demo eval bug

will-jl944 committed:
update hub serving doc

FlyingQianMM committed:
set encoding when open files in windows

FlyingQianMM committed:
set encoding when opening files in windows

will-jl944 committed:
update year in copyright

FlyingQianMM committed:
Merge pull request #1152 from will-jl944/develop_jf

Add restful

FlyingQianMM committed:
Merge pull request #1158 from will-jl944/hubsrv_jf

[WIP] Add hub serving

will-jl944 committed:
fix voc dataset empty bug

FlyingQianMM committed:
update version note

FlyingQianMM committed:
Merge pull request #1187 from FlyingQianMM/develop_qh

update note for paddlex2.0 not compatible with paddlex1.x

FlyingQianMM committed:
Merge pull request #1182 from will-jl944/develop_jf

fix voc dataset empty bug

FlyingQianMM committed:
update FAQ

FlyingQianMM committed:
Merge pull request #1188 from FlyingQianMM/develop_qh

update FAQ

will-jl944 committed:
set num_workers to 0 if negtive smaples are added

will-jl944 committed:
update ppdet

will-jl944 committed:
update pdpet

FlyingQianMM committed:
Merge pull request #1191 from will-jl944/develop_jf

Set num_workers to 0 when negative samples are added.

FlyingQianMM committed:
Revert "Set num_workers to 0 when negative samples are added."

FlyingQianMM committed:
Merge pull request #1192 from PaddlePaddle/revert-1191-develop_jf

Revert "Set num_workers to 0 when negative samples are added."

FlyingQianMM committed:
add deploy visualize in FAQ

FlyingQianMM committed:
Merge pull request #1194 from FlyingQianMM/develop_qh

add deploy visualize in FAQ

FlyingQianMM committed:
Update issue templates

FlyingQianMM committed:
update issue templates

FlyingQianMM committed:
Merge pull request #1196 from FlyingQianMM/develop_qh

update issue templates

will-jl944 committed:
fix resume checkpoint bug

will-jl944 committed:
Merge pull request #1197 from will-jl944/dev_bug_fix

fix resume checkpoint bug

yzl19940819 committed:
update_docs_1026

FlyingQianMM committed:
when loading a classifier or segmenter exporting by PaddleX 2.0.0rc, add 'net.' to each parameter name

FlyingQianMM committed:
when loading a classifier or segmenter exporting by PaddleX 2.0.0rc, add 'net.' to each parameter name

will-jl944 committed:
udpate ppdet

will-jl944 committed:
fix restful security issue

FlyingQianMM committed:
Merge pull request #1230 from will-jl944/dev_bug_fix

Fix RESTful security issue.

will-jl944 committed:
update ppdet

will-jl944 committed:
set batch_size to 1 if train rcnn with negative samples

will-jl944 committed:
force single card while training RCNN with negative samples

FlyingQianMM committed:
Merge pull request #1234 from will-jl944/dev_bug_fix

Fix RCNN negative sample training bug

yzl19940819 committed:
update doc

yzl19940819 committed:
update_doc

will-jl944 committed:
udpate restful

FlyingQianMM committed:
Merge branch 'develop' of https://github.com/PaddlePaddle/PaddleX into develop_qh

FlyingQianMM committed:
Merge pull request #1222 from FlyingQianMM/develop_qh

when loading a classifier or segmenter exporting by PaddleX 2.0.0rc, add 'net.' to each parameter name

FlyingQianMM committed:
add reference in _summarize

will-jl944 committed:
fix restful bug

will-jl944 committed:
fix restful bug

will-jl944 committed:
fix restful bug

will-jl944 committed:
2.0

yzl19940819 committed:
update doc_1109

FlyingQianMM committed:
Merge pull request #1237 from FlyingQianMM/develop_qh

add reference in _summarize

yzl19940819 committed:
update_1111

yzl19940819 committed:
update_1111-2

yzl19940819 committed:
update_1111-3

yzl19940819 committed:
update_1111-4

FlyingQianMM committed:
Merge pull request #1220 from yzl19940819/update_many_docs

update_docs_1026

FlyingQianMM committed:
Revert "update_docs_1026"

FlyingQianMM committed:
Merge pull request #1248 from PaddlePaddle/revert-1220-update_many_docs

Revert "update_docs_1026"

will-jl944 committed:
Merge branch 'develop' of https://github.com/PaddlePaddle/PaddleX into develop_jf

FlyingQianMM committed:
make major changes for docs

FlyingQianMM committed:
Merge pull request #1251 from FlyingQianMM/update_many_docs

make major changes for docs

FlyingQianMM committed:
fix the bugs in im_shape and scale_factor setting in c++ deployments

FlyingQianMM committed:
fix conflicts

FlyingQianMM committed:
Merge pull request #1253 from FlyingQianMM/develop_qh

fix the bugs in im_shape and scale_factor setting in c++ deployments

yzl19940819 committed:
change paddlex_whole.png

ucsk committed:
fix CenterCrop

FlyingQianMM committed:
Merge pull request #1259 from yzl19940819/update_docs

change paddlex_whole.png

will-jl944 committed:
Merge pull request #1260 from ucsk/develop

fix CenterCrop

will-jl944 committed:
Merge branch 'develop' of https://github.com/PaddlePaddle/PaddleX into develop_jf

yzl19940819 committed:
add activities

yzl19940819 committed:
fix_news

FlyingQianMM committed:
Merge pull request #1267 from yzl19940819/readme_add

add activities

FlyingQianMM committed:
do not copy tensorrt_dir/lib/myelin64_1.dll when using tensorrt>=8.0 with paddleinference

FlyingQianMM committed:
Merge pull request #1268 from FlyingQianMM/develop_qh

do not copy tensorrt_dir/lib/myelin64_1.dll when using tensorrt>=8.0 with Paddle Inference

will-jl944 committed:
Merge branch 'develop' of https://github.com/PaddlePaddle/PaddleX into develop_jf

FlyingQianMM committed:
Merge pull request #1241 from will-jl944/develop_jf

RESTful 2.0 ready

will-jl944 committed:
add picodet

will-jl944 committed:
check word_size before init_parallel to avoid warning

will-jl944 committed:
refine import

will-jl944 committed:
add picodet _compose_batch_transform

will-jl944 committed:
add picodet tutorial

will-jl944 committed:
add picodet

FlyingQianMM committed:
fix spelling mistake in manufacture sdk docs

FlyingQianMM committed:
Merge pull request #1277 from FlyingQianMM/develop_qh

fix spelling mistake in manufacture sdk docs

will-jl944 committed:
add picodet pretrained weights

will-jl944 committed:
accuracy checked

will-jl944 committed:
add picodet export

will-jl944 committed:
update ppdet

will-jl944 committed:
Update linux.md

will-jl944 committed:
polish code

FlyingQianMM committed:
Merge pull request #1278 from will-jl944/det_dev

Add PicoDet

will-jl944 committed:
install libopenblas-dev before install opencv

FlyingQianMM committed:
Merge pull request #1281 from will-jl944/develop_jf

Fix deployment compiling bug

will-jl944 committed:
update ppcls

will-jl944 committed:
update ppcls

FlyingQianMM committed:
Merge pull request #1282 from will-jl944/cls_dev

update ppcls

FlyingQianMM committed:
add cosine learning rate method for clas

will-jl944 committed:
add pplcnet

will-jl944 committed:
Update linux.md

will-jl944 committed:
add pplcnet tutorial

FlyingQianMM committed:
Merge pull request #1284 from will-jl944/cls_dev

Update ppcls; add PP–LCNet

will-jl944 committed:
add pplcnet doc

will-jl944 committed:
update ppdet

will-jl944 committed:
update ppdet

will-jl944 committed:
add picodet doc

will-jl944 committed:
avoid ppdet warning

will-jl944 committed:
update ppdet

will-jl944 committed:
Merge pull request #1286 from will-jl944/det_dev

Update ppdet; add picodet doc

will-jl944 committed:
update ppdet submodule

will-jl944 committed:
update ema

will-jl944 committed:
update picodet tutorial

will-jl944 committed:
Merge pull request #1288 from will-jl944/det_dev

update picodet tutorial

will-jl944 committed:
update version #

will-jl944 committed:
Merge pull request #1289 from will-jl944/det_dev

update version #

will-jl944 committed:
Merge pull request #1285 from will-jl944/cls_dev

add pplcnet doc

will-jl944 committed:
update requirements

will-jl944 committed:
update setup.py

will-jl944 committed:
Merge pull request #1290 from will-jl944/develop_jf

Ready to packup

will-jl944 committed:
silence ppseg warning

will-jl944 committed:
update setup.py

will-jl944 committed:
Update detection.md

will-jl944 committed:
picodet uses cosine optimizer by default

will-jl944 committed:
Merge pull request #1291 from will-jl944/det_dev

picodet uses cosine optimizer by default

will-jl944 committed:
Merge pull request #1292 from will-jl944/develop_jf

Update setup

will-jl944 committed:
update picodet tutorial trainign configuration

will-jl944 committed:
Merge pull request #1294 from will-jl944/develop_jf

update picodet tutorial trainign configuration

FlyingQianMM committed:
fix bugs for restful gui

FlyingQianMM committed:
Merge branch 'develop' of https://github.com/PaddlePaddle/PaddleX into develop_ui

will-jl944 committed:
disable picodet pruning

will-jl944 committed:
Merge pull request #1295 from will-jl944/develop_jf

disable picodet pruning

FlyingQianMM committed:
add picodet for restful

FlyingQianMM committed:
Merge branch 'develop' of https://github.com/PaddlePaddle/PaddleX into develop_ui

FlyingQianMM committed:
fix conflicts

FlyingQianMM committed:
Merge pull request #1297 from FlyingQianMM/develop_ui

fix bugs in restful api

will-jl944 committed:
update requirements

will-jl944 committed:
Merge pull request #1298 from will-jl944/develop_jf

update requirements

will-jl944 committed:
add demo for training w/ negative samples

will-jl944 committed:
Merge pull request #1300 from will-jl944/develop_jf

add demo for training w/ negative samples

will-jl944 committed:
Create first_meet.md

will-jl944 committed:
update detection dataset apis

will-jl944 committed:
Merge pull request #1301 from will-jl944/develop_jf

update detection dataset apis

FlyingQianMM committed:
add release note for 2.1.0

FlyingQianMM committed:
del restful gui in release note

FlyingQianMM committed:
Merge pull request #1302 from FlyingQianMM/develop_ui

add release note for 2.1.0

will-jl944 committed:
Update visualize.md

will-jl944 committed:
Update visualize.md

cjh3020889729 committed:
add __declspec(dllexport) instruction

fix the model_infer.cpp bug at windows

will-jl944 committed:
resume best acc and best acc epoch when resume training

FlyingQianMM committed:
Merge pull request #1319 from will-jl944/develop_jf

resume best acc and best acc epoch when resume training

will-jl944 committed:
update ppdet

will-jl944 committed:
training rcnn mdoel w/ negative sample supports bs>1

FlyingQianMM committed:
Merge pull request #1321 from will-jl944/develop_jf

Training rcnn mdoel w/ negative sample supports bs>1

FlyingQianMM committed:
Merge pull request #1318 from cjh3020889729/develop

Fix the DLL export problem of C # projects

linhandev committed:
[Enhancement] Add more info to voc x1<=x2 warning

Include the file path in 
```shell
"Bounding box for object {} does not satisfy x1 <= x2 and y1 <= y2, so this object is skipped. xml file: {}"
```
warning message. Should make it easier for user to locate error

linhandev committed:
[Enhancement] Add possible cause to x1<=x2 warning

Add a cause to this warning that is possible and may not be very easy to notice. May save users some time debugging the issue

linhandev committed:
Include bb value in warning message

Include bb value in warning message

FlyingQianMM committed:
Merge pull request #1337 from linhandev/develop

[Enhancement] Add more info to voc x1<=x2 warning

FlyingQianMM committed:
fix tile error in docs/data/annotation/classification.md

FlyingQianMM committed:
Merge pull request #1349 from FlyingQianMM/develop_ui

fix title error in docs/data/annotation/classification.md

FlyingQianMM committed:
fix cosine learning rate setting error for restful classification

FlyingQianMM committed:
Merge pull request #1357 from FlyingQianMM/develop_ui

fix cosine learning rate setting error for restful classification

Commit:
fix releases url link error

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/PaddlePaddle/PaddleX?pullRequest=1280) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/PaddlePaddle/PaddleX?pullRequest=1280) before we can accept your contribution.<br/>**7** out of **8** committers have signed the CLA.<br/><br/>:white_check_mark: cuicheng01<br/>:white_check_mark: changdazhou<br/>:white_check_mark: Bobholamovic<br/>:white_check_mark: dyning<br/>:white_check_mark: a31413510<br/>:white_check_mark: Zheng-Bicheng<br/>:white_check_mark: TingquanGao<br/>:x: Harryoung<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/PaddlePaddle/PaddleX?pullRequest=1280) it.</sub>

will-jl944 committed:
Update prediction.md

Fix invalid link.

FlyingQianMM committed:
fix wrong links in docs/gui

FlyingQianMM committed:
Merge pull request #1427 from FlyingQianMM/develop_qh

fix wrong links in docs/gui

an1018 committed:
add newExample

an1018 committed:
add newExample

an1018 committed:
update README

an1018 committed:
update README

FlyingQianMM committed:
Merge pull request #1440 from an1018/add_industry

add newExample

linhandev committed:
Delete .DS_Store

Remove useless file

FlyingQianMM committed:
fix docs/apis/prediction.md

FlyingQianMM committed:
Merge pull request #1480 from FlyingQianMM/develop_qh

fix docs/apis/prediction.md

FlyingQianMM committed:
fix VarifocalLoss import error

FlyingQianMM committed:
Merge pull request #1481 from FlyingQianMM/develop_qh

fix VarifocalLoss import error

FlyingQianMM committed:
rename pdx.model to pdx.det in example codes

FlyingQianMM committed:
Merge pull request #1482 from FlyingQianMM/develop_qh

rename pdx.model to pdx.det in example codes

FlyingQianMM committed:
Merge pull request #1464 from linhandev/patch-2

Delete .DS_Store

yzl19940819 committed:
update readme

FlyingQianMM committed:
Merge pull request #1493 from yzl19940819/readme

update readme

FlyingQianMM committed:
add add_negative_samples in docs

FlyingQianMM committed:
Merge pull request #1498 from FlyingQianMM/develop_qh

add add_negative_samples in docs/api

yzl19940819 committed:
update csharp docs

FlyingQianMM committed:
Merge pull request #1522 from yzl19940819/sharp

update csharp docs

FlyingQianMM committed:
clean fluid

FlyingQianMM committed:
update PaddleDetection submodule to clean fluid

FlyingQianMM committed:
Merge pull request #1526 from FlyingQianMM/develop_qh

clean fluid

angel-yi committed:
修改print为logging.info (#1558)

Co-authored-by: SI YI <18893465250@163.com>

GT-ZhangAcer committed:
修正文档一处描述错误 (#1499)

* Update detection.md

修正一处描述错误

* Update detection.md

linhandev committed:
Change instance segmentation demo dataset link (#1442)

[issue](https://github.com/PaddlePaddle/PaddleX/issues/1426)中说之前的垃圾实例分割数据集已经不再提供，将demo数据集链接更新为小度熊实例分割数据集

TrellixVulnTeam committed:
Adding tarfile member sanitization to extractall() (#1616)

jiangjiajun committed:
Update paddledetection.md

jiangjiajun committed:
Update paddledetection.md

jiangjiajun committed:
Update paddleclas.md

jiangjiajun committed:
Update paddleseg.md

jiangjiajun committed:
Update paddleseg.md

Commit:
Update QR code and qq group number

Commit:
Add new qq QR code file

shinoi2 committed:
Fix restful api bug, get json will a throw 415 Unsupported Media Type resp (#1707) (#1716)

[issue](https://github.com/PaddlePaddle/PaddleX/issues/1707)

juncaipeng committed:
[Doc] add introduction for new PaddleX (#1732)

Harryoung committed:
Update README.md

Harryoung committed:
Update README.md

Harryoung committed:
Update README.md

Harryoung committed:
Update README.md

Harryoung committed:
Update README.md

Harryoung committed:
Update README.md

cuicheng01 committed:
Update README.md

cuicheng01 committed:
Update README.md

dyning committed:
Update README.md

dyning committed:
Update README.md

changdazhou committed:
update requirements.txt

cuicheng01 committed:
Merge pull request #1757 from changdazhou/update_requirements

update requirements.txt

cuicheng01 committed:
paddlex3.0 first commit

cuicheng01 committed:
update README.md

cuicheng01 committed:
Merge pull request #1762 from cuicheng01/develop

X3.0 Dev first PR

cuicheng01 committed:
Update README.md

cuicheng01 committed:
Merge pull request #1763 from cuicheng01/develop

Update README.md

cuicheng01 committed:
support inference and wheel, add some docs

cuicheng01 committed:
Merge pull request #1764 from cuicheng01/develop

support inference and wheel, add some docs

cuicheng01 committed:
polish code & doc

cuicheng01 committed:
polish code and docs

cuicheng01 committed:
Merge pull request #1766 from cuicheng01/develop

polish code & doc

cuicheng01 committed:
polish docs and fix bugs

cuicheng01 committed:
fix copyright

cuicheng01 committed:
fix

cuicheng01 committed:
Merge pull request #1767 from cuicheng01/develop

polish docs and fix bugs

cuicheng01 committed:
polish

cuicheng01 committed:
Merge pull request #1769 from cuicheng01/develop

polish

cuicheng01 committed:
polish docs

cuicheng01 committed:
Merge pull request #1770 from cuicheng01/develop

polish docs

cuicheng01 committed:
update docs

cuicheng01 committed:
Merge pull request #1773 from cuicheng01/develop

update docs

cuicheng01 committed:
update docs

cuicheng01 committed:
Merge pull request #1774 from cuicheng01/develop

update docs

cuicheng01 committed:
update README

cuicheng01 committed:
Merge pull request #1775 from cuicheng01/develop

update README

Bobholamovic committed:
Add pipeline deployment docs

Bobholamovic committed:
Rename and polish

Bobholamovic committed:
Fix spacing and update table

cuicheng01 committed:
Merge pull request #1779 from Bobholamovic/deployment_docs

[Docs] Add Deployment Documentation

TingquanGao committed:
dbg:

1. use cpu to export model
2. automatically install dependent repo(s)

cuicheng01 committed:
Merge pull request #1783 from TingquanGao/dbg

dbg

a31413510 committed:
add different devices use guidande

cuicheng01 committed:
Merge pull request #1786 from a31413510/develop

add different devices use guidande

Zheng-Bicheng committed:
Update visualizer.py

cuicheng01 committed:
update README

TingquanGao committed:
update doc

cuicheng01 committed:
Merge pull request #1789 from Zheng-Bicheng/patch-1

Fixed the potential errors that may occur in PaddleX when the Pillow version is greater than 10.0.0

cuicheng01 committed:
Merge pull request #1790 from cuicheng01/develop

update README

cuicheng01 committed:
Merge pull request #1792 from TingquanGao/update

update doc
